coding-for-reusability/Exercise4/solution/Solution.tsx

```diff
@@ -0,0 +1,5 @@
+// function getLength<Type extends number[] | { length: number }>(
+//   list: Type
+// ): number {
+//   return list.length;
+// }
```

Comment:
In this example, CodeSandbox will show TypeScript’s `'getLength' is defined but never used` warning—that was the first thing that caught my attention when I opened the problem file. I wonder if we should “complete” the code so there aren’t any warnings like that.

Reply:
Sure, that's a simple enough fix